src/main/java/com/example/demo/CustomConfig.java

```diff
@@ -0,0 +1,18 @@
+package com.example.demo;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.Scanner;
+
+@Configuration
+@ComponentScan
```

Comment:
This is redundant, since spring-boot adds it by default(see. https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/autoconfigure/SpringBootApplication.html)